app/models/miq_ae_datastore.rb

```diff
@@ -256,4 +254,17 @@ def self.path_includes_domain?(path, options = {})
     nsd, = ::MiqAeEngine::MiqAePath.split(path, options)
     MiqAeNamespace.find_by_fqname(nsd, false) != nil
   end
+
+  def self.plugin_domain_names
+    datastores_for_reset
```

Comment:
Minor, but this method doesn't return domain names, it return domain objects.  So later when you do `plugin_domain_names.each { |domain| ...domain.datastores_path...` it reads funny because I would expect Strings to come out of a method with that name.

Reply:
Sure. May I just omit this method entirely since it's pretty superfluous?

Reply:
Yeah probably.